all/action.yml

```diff
@@ -0,0 +1,52 @@
+name: ODGen
+description: Run ODGen on your project
+
+inputs:
+  root:
+    description: The root of the project to analyze.
+    required: false
+    default: .
+  timeout:
+    description: The maximum duration of the analysis in seconds.
+    required: false
+    default: 300 # seconds, 5 minutes
+
+runs:
+  using: composite
+  steps:
+    - name: Scan for Command Injection vulnerabilities
+      uses: ericcornelissen/odgen-action@cde04479f42faa057bfad2ad733faa853cea4fc2
```

Comment:
Need to figure out a way to reference the current release here. Would be impossible to do with commit SHAs without having one release commit for the base action and one commit for the `/all` variant. Another option would be to just use version tags...

Reply:
Addressed in 4d6029ba77a7a60bc76fb932e85f44a2afbb8f90

Reply:
Re-opening at c429a3926ce16413844840d617ea313f3cbcefe6 because using `@v1` (as in 4d6029ba77a7a60bc76fb932e85f44a2afbb8f90) is incorrect because then using, say v1.0.3 when the latest release is v1.0.4, will result in the wrong version being used.

Reply:
Resolved with ab9738e3cbcfaa20a7d7690fa5d792690ba2261a